trangchu/sass/layouts/_content.scss

```
@@ -0,0 +1,32 @@
+$product-item-width: 48%;
```

```suggestion
$productItemWidth: 48%;
```
biến trong scss đặt theo kiểu camel case